packages/react/src/analytics/integrations/Vitorino/types/types.ts

```diff
@@ -2,6 +2,8 @@ import type {
   EventContext,
   IntegrationOptions,
 } from '@farfetch/blackout-analytics';
+import type { ForterOptions } from '../providers/Forter/types';
+import type { RiskifiedOptions } from '../providers/Riskified/types';
 
 export type EventsMapper = Record<
```

Comment:
There are some types here like `Fields`, `Network` and `Environment` that are not being exported, but should as they are being used in other public exported types.

Reply:
Added export for this types

Reply:
Sorry to comment again on this, but could you add a more specific name for these types? `Fields`, `Network` and `Environment` are too generic and these types are specific to Guardian. Also, there is an `EnvironmentTypes` type as well which is being used to compare to the value of the `environment` option in Guardian which is typed as `Environment`. My question is can't it reuse the `EnvironmentTypes` and drop `Environment` altogether?

Reply:
Changed to use EnvironmentTypes instead of previous Environment

Reply:
`Fields` and `Network` are still using the same names and they are too generic to be exported as is. From what I can see they are used only inside the `Config` type (which is also too generic). It should be something like this:

`Config` -> `GuardianConfig` (also double check if Config is being used because as far as I can see it is not being used anywhere.
`Fields` -> `GuardianSensitiveFields`
`Network` -> `GuardianNetworkSettings`